apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -65,6 +65,15 @@ Page {
                 //: Label for combobox that sets search engine used in browser
                 //% "Search with"
                 label: qsTrId("settings_browser-la-search_with")
+                leftMargin: 2*Theme.horizontalPageMargin + icon.width
+                contentHeight: Theme.itemSizeMedium
+
+                Icon {
+                    id: icon
```

Comment:
There is a other icon in this file, 'permissionIcon', I think it would be better to rename it to more specific

Reply:
@atatarov  added title `searchIcon`, which matches the `searchEngine` element